tests/integration/test_validate_synced_files_existance.py

```diff
@@ -0,0 +1,98 @@
+    [
+        (
+            """
+            {
+                "config_file_path": "packit.json",
+                "dist_git_base_url": "https://packit.dev/",
+                "downstream_package_name": "packit",
+                "upstream_ref": "last_commit",
+                "upstream_package_name": "packit_upstream",
+                "create_tarball_command": ["commands"],
+                "allowed_gpg_keys": ["gpg"],
+                "dist_git_namespace": "awesome",
+                "synced_files": [{ "src": "a.md", "dest": "aaa.md" }, "a_dir"]
+            }
+            """,
+            "packit.json is valid and ready to be used",
+        ),
+        (
+            """
+            {
+                "config_file_path": "packit.json",
+                "dist_git_base_url": "https://packit.dev/",
+                "downstream_package_name": "packit",
+                "upstream_ref": "last_commit",
+                "upstream_package_name": "packit_upstream",
+                "create_tarball_command": ["commands"],
+                "allowed_gpg_keys": ["gpg"],
+                "dist_git_namespace": "awesome",
+                "synced_files": ["a.md", "b.md", "c.txt", "a_dir"]
+            }
+            """,
+            "The following path does not exist: c.txt",
+        ),
+    ],
+    ids=[
+        "none_missing",
+        "one_missing",
+    ],
+)
+def test_validate_synced_files_one_missing(tmpdir, raw_package_config, expected_output):
+    with cwd(tmpdir):
+        Path("packit.json").write_text(raw_package_config)
+        Path("packit.spec").write_text("hello")
+        Path("a.md").write_text("a")
+        Path("b.md").write_text("b")
+        Path("./a_dir").mkdir()
+
+        output = PackitAPI.validate_package_config(Path("."))
+        assert expected_output in output
+
+
+@pytest.mark.parametrize(
+    "raw_package_config,expected_output",
+    [
+        (
+            """
+             {
+                 "config_file_path": "packit.json",
+                 "dist_git_base_url": "https://packit.dev/",
+                 "downstream_package_name": "packit",
+                 "upstream_ref": "last_commit",
+                 "upstream_package_name": "packit_upstream",
+                 "create_tarball_command": ["commands"],
+                 "allowed_gpg_keys": ["gpg"],
+                 "dist_git_namespace": "awesome",
+                 "synced_files": ["a.md", "b.md", "c.txt"]
+             }
+             """,
+            "The following paths do not exist: b.md, c.txt",
+        ),
+    ],
+    ids=[
+        "two_missing",
+    ],
+)
+def test_validate_synced_files_multiple_missing(
+    tmpdir, raw_package_config, expected_output
+):
+    with cwd(tmpdir):
+        Path("packit.json").write_text(raw_package_config)
+        Path("packit.spec").write_text("hello")
+        Path("a.md").write_text("a")
+
+        output = PackitAPI.validate_package_config(Path("."))
+        assert expected_output in output
```

Comment:
Would it be possible to merge all three use cases into one test case? (The tests are parametrised and the code is very similar so it shouldn't be a problem I hope.)